sqlgen/db.go

```diff
@@ -48,8 +51,9 @@ type DynamicLimit struct {
 
 func NewDB(conn *sql.DB, schema *Schema) *DB {
 	db := &DB{
-		Conn:   conn,
-		Schema: schema,
+		Conn:           conn,
+		Schema:         schema,
+		panicOnNoIndex: false,
```

Comment:
could let the `false` be implicit, but I sort of prefer being explicit on this type of stuff.